### PR TITLE
Add game timer

### PR DESCRIPTION
**Pull request by smculliton**
Add game timer functionality

### CODE REVIEW

File: lib/timer.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'time'
```
Comment (bflanagan138) [inline lib/timer.rb:1]: Good job with creating separate timer class rather than trying to fit it inside the game class.

File: lib/timer.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    def start_timer 
+        @start = Time.now.to_i
+    end
+
+    def stop_timer
+        @stop = Time.now.to_i
+    end
+    
+    def elapsed_time
+        @stop - @start
```
Comment (bflanagan138) [inline lib/timer.rb:21]: To create our elapsed_time method, we figured out that Time.now.to_i returned an integer value, we were able to take that value and, using the start_timer and stop_timer methods get a difference that indicated how many seconds lapsed.

File: lib/timer.rb (hunk 1 above)
Comment (smculliton) [reply lib/timer.rb:1]: thanks!

**Review by bflanagan138 — APPROVED** [review]
Super stoked on how the Timer class came out!

**Comment by bflanagan138** [comment]
Really excited about how this game has come together!!!

**Pull request merged**